DockDoor/Views/Settings/DockPreviewsView.swift

```diff
@@ -0,0 +1,227 @@
+import Defaults
+import SwiftUI
+
+// Add enum for preview size
+private enum DockPreviewSize: String, CaseIterable, Identifiable {
```

Comment:
Move this to consts.swift where the other declarations are of settings configurations